_sass/components/_article-table.scss

```diff
@@ -45,6 +45,10 @@ $max-columns: 10;
     border-top: 1px solid $gray-light;
     line-height: 1.3;
     padding: $base-padding-small;
+
+    &.article_table-note {
```

Comment:
Love this approach to making a class that more widely usable. @brentryanjohnson and @master12 if we went this way, it would mean we can change the markup on tables that use these note styles (seen here, built with highly-specific markup that I don't know the backstory on: https://revenuedata.doi.gov/how-it-works/reconciliation/2016/ ) Are you interested in that as a follow-up task? How would you like to approach this, since it was a bonus commit, but not specifically required for the issues this PR addresses?

Reply:
@jenniferthibault I think we should merge it in and create a new issue to generalize the markup for the notes. We aren't likely to maintain this particular content (Reconciliation), since it was part of the EITI process, but that's even more reason to take the approach proposed in this PR.

Reply:
That makes sense to me. What are your thoughts @master12 ?